File: app/src/main/java/com/example/myapplication/Roslina1.java
package com.example.myapplication;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import androidx.annotation.Nullable;
import androidx.appcompat.app.AppCompatActivity;

public class Roslina1 extends AppCompatActivity {
    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.roslina_opis);

        Button powrot = (Button)findViewById(R.id.Powrot);
        powrot.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(Roslina1.this, MainActivity.class));
            }
        });
    }

}

File: app/src/main/java/com/example/myapplication/Roslina1.kt
package com.example.myapplication

import android.os.Bundle
import android.view.View
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class Roslina1 : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.roslina_opis)
        val powrot = findViewById<View>(R.id.Powrot) as Button
        powrot.setOnClickListener { finish() }
    }
}